packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -740,6 +777,31 @@ const QueryBuilderGridResult = observer(
     const { executionResult, queryBuilderState } = props;
 
     const resultState = queryBuilderState.resultState;
+    const isAdvancedModeEnabled = queryBuilderState.isAdvancedModeEnabled;
+    const colDefs = executionResult.result.columns.map(
```

Comment:
for this could we create two separate functions for advanced and simplle definitions. would be cleaner than having flags in each config